code/datums/datum.dm

```diff
@@ -9,6 +9,21 @@
 	/// Datum level flags
 	var/datum_flags = NONE
 
+	/**
+	  * Any datum registered to receive observations from this datum is in this list
+	  *
+	  * Lazy associated list in the structure of `observation:registree/list of registrees`
+	  */
+	var/list/observations
+	/// Lazy associated list in the structure of `observations:proctype` that are run when the datum receives that observation
+	var/list/list/datum/callback/observation_procs
+	/**
+	  * Is this datum capable of sending observations?
+	  *
+	  * Set to true when a observation has been registered
+	  */
+	var/observation_enabled = FALSE
```

Comment:
this is redundant, remove it. The observations list being null or not is enough to determine if it cares about events

Reply:
remove this